es/index.html

```diff
@@ -0,0 +1,502 @@
+          </p>
+
+          <a
+            href="https://ethanmarcotte.com/wrote/accessibility-is-not-a-feature/"
+            target="_blank"
+            >Más detalles
+            <span class="visually-hidden"
+              >sobre La accesibilidad es una funcionalidad: lo haces una vez y listo</span
+            ></a
+          >
+        </section>
+
+        <section class="myth">
+          <h2 class="myth__title">
+            La accesibilidad se puede arreglar ofreciendo una versión especial
+            para personas con discapacidad
+          </h2>
+
+          <p>
+            Esto siempre estuvo mal. Muchos equipos de desarrollo
+            han <strong>aprendido por las malas</strong> que mantener diferentes versiones
+            y el contenido actualizado de un sitio web <strong>es mucho más costoso</strong>
+            que crear un sitio accesible para todas las personas.
+          </p>
+        </section>
+
+        <section class="myth">
+          <h2 class="myth__title">
+            Las personas con discapacidad no usan la web
+          </h2>
+
+          <p>
+            Eso es incorrecto. De hecho, en algunos casos, internet es
+            <strong>la única forma</strong> cómo las personas con discapacidad pueden comunicarse
+            con el mundo. Además, <strong>confían en la tecnología</strong> para poder percibir y comunicarse.
+          </p>
+
+          <a href="https://www.w3.org/WAI/people-use-web/" target="_blank"
+            >Más detalles
+            <span class="visually-hidden"
+              >sobre Las personas con discapacidad no usan la web</span
+            ></a
+          >
+        </section>
+
+        <section class="myth">
+          <h2 class="myth__title">Las personas ciegas no miran películas</h2>
+
+          <p>
+            Definitivamente lo hacen. Con la ayuda de
+            <strong>la audiodescripción</strong> pueden ver y escuchar cualquier contenido
+            multimedia.
+          </p>
+
+          <a href="https://www.w3.org/WAI/media/av/description/" target="_blank"
+            >Más detalles
+            <span class="visually-hidden"
+              >sobre Las personas ciegas no miran películas</span
+            ></a
+          >
+        </section>
+
+        <section class="myth">
+          <h2 class="myth__title">
+            La accesibilidad solo puede ser probada por personas con discapacidad
+          </h2>
+
+          <p>
+            Bueno, por lo general las personas con alguna discapacidad son las mejores para probar
+            - ya que todo el tiempo usan tecnologías asistivas. Sin embargo,
+            <strong>cualquiera puede aprender</strong> a probar la accesibilidad
+            de los sitio webs.
+          </p>
+
+          <a href="https://www.w3.org/WAI/test-evaluate/" target="_blank"
+            >Más detalles
+            <span class="visually-hidden"
+              >sobre Accessibility can only be tested by disabled people</span
```

Comment:
@pazguille could you add translation for `Accessibility can only be tested by disabled people`?